motion_computation/src/motion_computation_worker.cpp

```diff
@@ -35,6 +35,10 @@ void MotionComputationWorker::predictionLogic(
   carma_perception_msgs::msg::ExternalObjectList sensor_list;
   sensor_list.header = obj_list->header;
 
+  if (!obj_list || obj_list->objects.empty())
+  {
+    return;
+  }
```

Comment:
Nit: Could we add a line of whitespace after this code block before the for-loop just to improve readability at a quick glance?

Reply:
Added